astropy/coordinates/funcs.py

```diff
@@ -149,11 +152,23 @@ def get_sun(time):
     from .builtin_frames import GCRS
 
     earth_pv_helio, earth_pv_bary = erfa.epv00(time.jd1, time.jd2)
-    x = -earth_pv_helio[..., 0, 0] * u.AU
-    y = -earth_pv_helio[..., 0, 1] * u.AU
-    z = -earth_pv_helio[..., 0, 2] * u.AU
+
+    # We have to manually do abberation because we're outputting directly into
```

Comment:
Would it make sense to extract these aberration formulae into a helper function?
Might not be worth exposing in `__all__`, but could come in handy in other places in `astropy.coordinates` or other packages that use `astropy.coordinates`?
Just a thought ... I'm not sure.